src/pages/edit/index.js

```diff
@@ -0,0 +1,236 @@
+import { Route } from "react-router-dom";
+import Card from "../../components/card";
+import TextInput from "../../components/form/textInput";
+import "./style.css";
+import "../../styles/index.css";
+import LockIcon from "../../assets/icons/locIcon";
+
+const hrStyle = {
+  backgroundColor: "e6ebf5",
+  height: 1,
+};
```

Comment:
Let's move this to a css class instead of a js object

Reply:
Changed to CSS class as requested.